scipy/signal/tests/test_signaltools.py

```diff
@@ -846,15 +951,20 @@ def gen_oa_shapes_eq(sizes):
             if a >= b]
 
 
+
+@skip_xp_backends(cpu_only=True, exceptions=['cupy'])
+@skip_xp_backends("jax.numpy", reason="fails all around")
```

Comment:
Should we offer delegation to a function that doesn't pass any of our tests?

Reply:
We don't --- there's a whitelist of jax functions, and oaconvolve is not included. It works on torch though, so I'd keep skipping it on jax.numpy and it for a follow-up.